app/controllers/snps_controller.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 class SnpsController < ApplicationController
-  helper_method :sort_column, :sort_direction
   before_filter :find_snp, except: [:index, :json,:json_annotation]
 
   def index
-    @snps = Snp.order(sort_column + ' ' + sort_direction)
+    params.delete(:sort)
+    params.delete(:direction)
```

Comment:
Deleting the params is likely not necessary. By not reading the params, you are already ignoring them.

Reply:
I thought the same! But playing around with it on my local system the table kept on getting sorted depending on the params in the URL I set manually, I didn't think that was possible

Reply:
Me neither... :thinking:

Reply:
it is a superugly solution and I welcome better ideas :D

Reply:
Wait, why are these back? With my change to `Snp.order()` this really isn't needed as the parameters are a) never parsed as the helper methods are gone and b) the order command now directly states that the ordering is performed on ranking?

Reply:
You had to delete them earlier as you didn't change the `order` call, but my commits in this PR changed that ☺️

Reply:
Hmm i didn't put them back in, must've been github merging commits?